.github/workflows/build-app.prod.yml

```diff
@@ -0,0 +1,24 @@
+on: push
+
+jobs:
+  build:
+    name: Build application
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v2
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: "17"
+          cache: npm
+
+      - name: 🧰  Install
+        run: npm ci --production-only
+
+      - name: 📦  build
+        run: npx nx build team-awesome-web
```

Comment:
You _could_ put these in the `package.json` file like we did with `npm run dev`.